packages/block-library/src/buttons/style.scss

```diff
@@ -18,6 +18,16 @@
 	}
 }
 
+.wp-block-buttons.alignleft .wp-block-button {
```

Comment:
do we need this since it's the default style if we don't have alignments?

Reply:
Yes because the default in RTL settings is right aligned.

Reply:
But wouldn't the style be flipped in the RTL mode?

Reply:
No, the alignment buttons are the specific exception because the class names are explicitly set.

Reply:
Does this mean we need to wrap these with `/*!rtl:begin:ignore*/` ?

Reply:
MAYBE.

Yes. Actually it does. On it.